src/analyzing/CFG_build.ml

```diff
@@ -73,7 +73,8 @@ type state = {
    * could not happen.
    *)
   lambdas : (name, IL.function_definition) Hashtbl.t;
-  (* If a lambda is never used, we just insert its CFG at declaration site. *)
+  (* If a lambda is never used, we just insert its CFG at the end of
+     its parent function. *)
   unused_lambdas : (name, nodei * nodei) Hashtbl.t;
```

Comment:
Better to clean this up I think, and remove `nodei * nodei`, this would be a set now instead. That pair of nodes was stored there just to know where to insert the CFG of the lambda.

Reply:
Done.